### PR TITLE
Documentation: Update testing doc

### PR DESCRIPTION
**Pull request by noaos**
Update testing.md to properly reflect recent changes to the test suite
execution.

Fixes: d6c1c233 ('tests: Fix test locating process')
Signed-off-by: Noa Osherovich <noaos@mellanox.com>

### CODE REVIEW

File: Documentation/testing.md [hunk 1]
```diff
@@ -70,7 +75,7 @@ that failed.
 
 Tests can also be executed in verbose mode:
 ```
-$ python3 run_tests.py -v
+$ python3 /usr/share/doc/rdma-core-26.0>/tests/run_tests.py -v
```
Comment (Kamalheib) [inline Documentation/testing.md]: extra ">"

File: Documentation/testing.md [hunk 2]
```diff
@@ -49,16 +49,21 @@ valid RDMA resources for proper functioning.
...
 ```
-cd /usr/share/doc/rdma-core-26.0/tests
-python3 run_tests.py
+python /usr/share/doc/rdma-core-<version>/tests/run_tests.py
 ```
+#### Execution output
 Output will be something like:
 ```
-$ python3 run_tests.py
+$ ./build/bin/run_tests.py
```
Comment (juhlee-microsoft) [inline Documentation/testing.md]: Recent RHEL8 and CentOS8 show no longer python in the release image. How can we make it satisfy this line? Is there test setup requirements?

Comment (noaos) [reply Documentation/testing.md]: From what I understand (link  below), /usr/bin/python has been removed from rhel8, thus the python alias. You should be able to use python3. Running from build/bin should be the same. Running from fresh installation will probably be something like:
python3 /usr/share/<>/run_tests.py

If your issue is different, please provide more details.

https://developers.redhat.com/blog/2019/05/07/what-no-python-in-red-hat-enterprise-linux-8/

File: Documentation/testing.md (hunk 1 above)
Comment (noaos) [reply Documentation/testing.md]: Fixed, thanks.

File: Documentation/testing.md (hunk 2 above)
Comment (noaos) [reply Documentation/testing.md]: Also, python3 is defined as a requirement in RH's spec file of rdma-core.

Comment (juhlee-microsoft) [reply Documentation/testing.md]: Ok, then this should not block the user. Thanks.

**Pull request merged**